### PR TITLE
Experimental/unified design

### PR DESCRIPTION
**Pull request by avullo**
This is introducing RDF writers for features (non ensembl) and corresponding xrefs.
Includes a minor change on the way taxonomy data is described.

### CODE REVIEW

File: modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+}
+
+=head2 coord_system_name
+
+=cut
+
+sub coord_system_name {
+  my ($self, $object) = @_;
+  
+  return $object->{coord_system}{name};
```
Comment (lairdm) [inline modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm]: I know it's a trade off between speed and safety, and do we want to have it blow up early or give a better warning when it blows up. Do we want to blindly try to grab an element from a hash?

Comment (avullo) [inline modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm]: You're right, care should be taken at one point or another. Nevertheless, in this particular case, I think we're safe to favor speed as this emanates from a direct contract stipulated with Dan S. He's already promptly informed of an intended change to the underlying data structure.

Comment (lairdm) [inline modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm]: Sorry, another change to come? Or are we good to merge now?

Comment (avullo) [inline modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm]: Not another one, I believe we're good to go!

Comment (avullo) [inline modules/Bio/EnsEMBL/IO/Translator/BulkFetcherFeature.pm]: Anyway, It's good you've highlighted this point. Once the changes are in place and the pipeline runs with ensembl-io, I will graduallyt introduce updates to make it more robust and safe, monitoring performance at the same time.

**Pull request merged**